plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/SourceAttacherJob.java

```diff
@@ -54,37 +56,36 @@ protected IStatus run(IProgressMonitor monitor) {
     try {
       IClasspathContainer container = JavaCore.getClasspathContainer(containerPath, javaProject);
       if (container instanceof LibraryClasspathContainer) {
```

Comment:
Shouldn't the condition be negated? If so, and tests are still passing can you add one that fails on this before fixing it?